src/app/token/TokenPage.js

```diff
@@ -33,15 +33,13 @@ const mapDispatchToProps: DispatchProps = {
   complete,
 }
 
-type Props = {|
-|} & StateProps & DispatchProps
+type Props = {||} & StateProps & DispatchProps
 
 type State = {|
   isModalOpen: boolean,
 |}
 
 class TokenPage extends Component<Props, State> {
-
```

Comment:
please stop doing this. I told you like months ago that I add this empty line sometimes to make code more readable

Reply:
Fixed.